### PR TITLE
Refactor 1

### PR DESCRIPTION
**Pull request by artemd24**
1) Remove unnecessary indent,
2) Change handlers to POST /users - registration, GET /users/{id} - receiving information,
3) Change version

### CODE REVIEW

File: src/main/java/com/wine/to/up/user/service/controller/UserController.java [hunk 1]
```diff
@@ -10,19 +10,18 @@
...
 
-    @GetMapping("/get")
+    @GetMapping("/{id}")
     public ResponseEntity<UserDto> findUserByID(@RequestParam Long id) {
         UserDto user = userService.getById(id);
         return new ResponseEntity<>(user, HttpStatus.OK);
     }
 
-    @PostMapping("/register")
+    @PostMapping("")
```
Comment (Kot1que) [inline src/main/java/com/wine/to/up/user/service/controller/UserController.java]: Можно оставить просто `@PostMapping`

File: src/main/java/com/wine/to/up/user/service/controller/UserController.java [hunk 2]
```diff
@@ -10,19 +10,18 @@
...
-@RequestMapping("/user")
+@RequestMapping("/users")
 @Slf4j
-
 public class UserController {
     public final UserService userService;
 
-    @GetMapping("/get")
+    @GetMapping("/{id}")
     public ResponseEntity<UserDto> findUserByID(@RequestParam Long id) {
```
Comment (Kot1que) [inline src/main/java/com/wine/to/up/user/service/controller/UserController.java]: `@RequestParam` -> `@PathVariable`

RequestParam: example.com/users?id=123
PathVariable: example.com/users/123

**Review by Kot1que — CHANGES_REQUESTED** [review]

File: src/main/java/com/wine/to/up/user/service/controller/UserController.java (hunk 1 above)
Comment (artemd24) [reply src/main/java/com/wine/to/up/user/service/controller/UserController.java]: Done

File: src/main/java/com/wine/to/up/user/service/controller/UserController.java (hunk 2 above)
Comment (artemd24) [reply src/main/java/com/wine/to/up/user/service/controller/UserController.java]: Done

**Review by Kot1que — APPROVED** [review]

**Pull request merged**